### PR TITLE
Check link whitelist status properly

### LINKED ISSUE #709: Link whitelist doesn't seem to work
I have setup whitelisted links in #weest. Non-subs also get timed out for posting links. My assumption would be that whitelisting a link would have non-subs not be timed out...unless someone can better explain what the whitelist should do - I think it should be improved to include things like this. Would also be cool if it supported wildcards like `*twitch.tv*`

### PR DESCRIPTION
**Pull request by pajlada**
Fixes #709 

Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -125,7 +126,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` when updating to this version!
 
 - Breaking: pajbot now uses PostgreSQL instead of MySQL as its supported
-  database engine. It is not possible to continue to use MySQL.  
+  database engine. It is not possible to continue to use MySQL.
```
Comment (RAnders00) [inline CHANGELOG.md]: why was this changed

Comment (pajlada) [reply CHANGELOG.md]: my linter removed that, I'll fix it NaM

**Comment by alazymeme** [comment]
pajaApproved
